source/Jobbr.Tests/UnitTests/Storage/ExtensionMethodsTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Jobbr.Server.Storage;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Jobbr.Tests.UnitTests.Storage
+{
+    [TestClass]
+    public class ExtensionMethodsTests
+    {
+        [TestMethod]
+        public void CloneNullObject_ShouldNotThrow()
+        {
+            // Arrange
+            object data = null;
+
+            // Act
+            var dataClone = ExtensionMethods.Clone(data);
+
+            // Assert
+            Assert.IsNull(dataClone);
+        }
+
+        [TestMethod]
+        public void CloneObject_ShouldCreateDeepClone()
+        {
+            // Arrange
+            var data = new TestClass("reference", 5);
+
+            // Act
+            var dataClone = ExtensionMethods.Clone(data);
+
+            // Assert
+            Assert.IsNotNull(dataClone);
+
+            Assert.AreEqual(data.Key, dataClone.Key);
+
+            Assert.AreEqual(data.Value, dataClone.Value);
+
+            Assert.AreNotEqual(data, dataClone);
+        }
+
+        [TestMethod]
+        [DataRow(1)]
+        [DataRow(1.45)]
+        [DataRow("test")]
+        public void CloneValueTypes_ShouldReturnCopy(object value)
+        {
+            // Arrange
+
+            // Act
+            var clone = ExtensionMethods.Clone(value);
+
+            // Assert
+            Assert.AreEqual(clone, value);
+        }
+
+        [TestMethod]
+        public void CloneList_ShouldDeepCloneAllEntries()
+        {
+            List<TestClass> data = new() { new("value-1", 1), new("value-2", 2), new("value-3", 3) };
```

Comment:
Always use `var` and if the type is needed rather have it on the right side with the `new` calls